elements/lisk-cryptography/test/bls_lib/lib.spec.ts

```diff
@@ -144,6 +144,46 @@ describe('bls_lib', () => {
 	});
 
 	describe('blsFastAggregateVerify', () => {
+		describe('special invalid public key', () => {
```

Comment:
instead of adding test manually, I think you can add fixture and include it in the test below?